deploy/utils/README.md

```diff
@@ -20,7 +20,7 @@ This includes:
   - `serviceaccount.yaml` — ServiceAccount `dynamo-sa` for benchmarking and profiling jobs
   - `role.yaml` — Role `dynamo-role` with necessary permissions
   - `rolebinding.yaml` — RoleBinding `dynamo-binding`
-  - `pvc.yaml` — PVC `dynamo-pvc` for storing profiler results and configurations
+  - `pvc.yaml` — PVC `planner-pvc` for storing profiler results and configurations
```

Comment:
Can we keep the PVC name as is? And use it for general k8s file storage within dynamo